exercises/01-composite-ui/before/Divergent.Frontend/app/branding/orders/ordersView.html

```diff
@@ -12,4 +12,8 @@
         </li>
     </ul>
 
+    <div ng-show="ctrl.loadingData" class="data-load">
+        Data is loading....
```

Comment:
Perhaps we could have a spinner here instead (with "Loading" alt text)? E.g.

![ajax-loader](https://user-images.githubusercontent.com/677704/28826751-19298bfa-76c3-11e7-99b9-804c26b49922.gif)